src/functions/node.php

```diff
@@ -21,3 +25,51 @@ function node_view($node, $view_mode = 'full', $langcode = null)
       ->getViewBuilder('node')
       ->view($node, $view_mode, $langcode);
 }
+
+function node_type_get_name(EntityInterface|string $node): string|false
+{
+    if ($node instanceof EntityInterface) {
+        return $node->bundle();
+    } else {
+        $types = NodeType::loadMultiple();
+        return isset($types[$node]) ? $node : false;
+    }
+}
+
+function node_type_load(string $name): ?NodeTypeInterface
+{
+    return NodeType::load($name);
+}
+
+/**
+ * @param array{
+ *   type?: string,
+ *   name?: string,
+ *   base?: string,
+ *   description?: string,
+ *   help?: string,
+ *   custom?: int,
+ *   modified?: int,
+ *   locked?: int,
+ *   disabled?: int,
+ *   is_new?: int,
+ *   has_title?: int,
+ *   title_label?: string
+ * } $info
+ */
+function node_type_set_defaults(array $info = []): NodeTypeInterface
+{
+    return NodeType::create($info);
+}
+
+function node_type_save(NodeTypeInterface $info): int
+{
+    return $info->save();
+}
+
+function node_type_delete(string $type): void
+{
+    $type = NodeType::load($type);
+    assert($type instanceof NodeTypeInterface);
+    $type->delete();
```

Comment:
When reviewing https://git.drupalcode.org/project/drupal/-/blob/7.x/modules/node/node.module#L652-661 it looks like it would always work, even if given an invalid node type ID.

Can we convert `assert` to an `if` statement block